desktop/src/main/java/bisq/desktop/main/market/offerbook/OfferBookChartView.java

```diff
@@ -421,9 +427,47 @@ private void updateChartData() {
 
         seriesBuy.getData().addAll(model.getBuyData());
         seriesSell.getData().addAll(model.getSellData());
+
+        // Before adding the points to the chart, set a custom node used for showing a label on hover
```

Comment:
Instead of a comment, I would prefer the nicely named method. Comments tend to age and becoming inconsistent with the code. Best documentation is the code itself.